doc/nrf/gs_installing.rst

```diff
@@ -280,7 +280,7 @@ To clone the repositories, complete the following steps:
    * To work with a branch, the identifier is the branch name (for example, ``main`` to work with the latest state of development).
    * To work with a specific state, the identifier is the SHA (for example, ``224bee9055d986fe2677149b8cbda0ff10650a6e``).
 
-#. From the command line, initialize west with the revision of the |NCS| that you want to check out, replacing *nRFConnectSDK_revision* with the identifier:
+#. From the command line, ``cd`` into the :file:`ncs` folder and initialize west with the revision of the |NCS| that you want to check out, replacing *nRFConnectSDK_revision* with the identifier:
```

Comment:
Consider also changing

` This folder will hold all |NCS| repositories.`

To `This folder will become your workspace, holding all |NCS| files and sub-repositories.`


For me at least, that wmight have clarified things just a bit. When I followed these instructions for the first time, I wasn't completely clear on whether there would be a workspace created inside ncs, or whether ncs itself would become the workspace. It turned out that ncs itself becomes the workspace, and that's not what I personally wanted, so I wound up having to redo it inside another folder inside my ncs folder.

Reply:
Good point. But I don’t want to overcomplicate this change, I will leave it to tech writers to follow-up

Reply:
cc @peknis for a follow-up PR